src/benchmark/tpch_benchmark.cpp

```diff
@@ -110,7 +110,8 @@ int main(int argc, char* argv[]) {
 
   // TODO(leander): Enable support for queries that contain multiple statements requiring execution
   if (config->enable_scheduler) {
-    Assert(std::find(query_ids.begin(), query_ids.end(), opossum::QueryID{15}) == query_ids.end(),
+    // QueryID{14} represents TPC-H query 15 because of the way we generate and store the queries
```

Comment:
That sounds cryptic. "QueryID{14} represents TPC-H query 15 because we use 0 indexing", I suppose?

Reply:
Right, that makes sense!